src/stylesheets/elements/_buttons.scss

```diff
@@ -279,29 +322,29 @@
   Circular Buttons
 --------------------*/
 // Utility buttons only include an icon
-
+// DEPRECATED: Use .sds-button--circle
```

Comment:
is this still depreciated?

Reply:
yes, we are sort of turning the tables on this. It is still used on the Actions menu but that will have to be addressed there so in the mean time it is deprecated